07-eval/client_eval_test.py

```diff
@@ -4,66 +4,47 @@
 #
 import asyncio
 import os
-import shutil
 
+from delayed_assert.delayed_assert import assert_all, expect
```

Comment:
interesting. just for my info, delayed_assert?

Reply:
Yeah - sorry if it wasn't clear, this is to allow pytest to continue passed an assertion aka "soft assertion", for evals it's probably good to see all assertion results together